phys2bids/tests/test_physio_obj.py

```diff
@@ -101,11 +101,12 @@ def test_BlueprintInput():
 
     # Tests rename_channels
     new_names = ['trigger', 'time', 'lindt']
-    blueprint_in.rename_channels(new_names.copy())
+    blueprint_in.rename_channels(new_names)
+    test_index = blueprint_in.return_index(1)
     assert blueprint_in.ch_name == ['time', 'trigger', 'lindt']
+    assert test_index[0] is not test_trigger
 
     # Tests return_index
-    test_index = blueprint_in.return_index(1)
     assert (test_index[0] == test_trigger).all()
```

Comment:
aren't 107 and 110 uncompatible?

Reply:
you just need to move 107 down to 110

Reply:
That's not my point, now your line is failing as you said it should. What I am raising as question is why  do you propose an assert (107) that is contradictory to one we already have (110)

Reply:
Ah, I see. `is` isn't the same as `==`. `==` should check that the arrays contain equivalent values (although `np.array_equal` would be cleaner), while `is` should determine if the variables are referring to the same object.

Reply:
ah, I see your point